stripe.go

```diff
@@ -72,13 +78,15 @@ func (a *AppInfo) formatUserAgent() string {
 type Backend interface {
 	Call(method, path, key string, body *form.Values, params *Params, v interface{}) error
 	CallMultipart(method, path, key, boundary string, body io.Reader, params *Params, v interface{}) error
+	SetMaxNetworkRetry(maxNetworkRetry int)
```

Comment:
It seems that using "retries" instead of "retry" would make this more fluid, and also be consistent with `MaxNetworkRetries` in `BackendConfiguration` below. Any reason you went with the singular version of the word here?